com/course/zenkouv/src/test/java/test/webDriverTest/OnlinerTest.java

```diff
@@ -0,0 +1,42 @@
+package test.webDriverTest;
+
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import webDriver.factory.WebDriverFactory;
+import webDriver.pageObject.onliner.*;
+
+public class OnlinerTest {
+
+    private final String url = "https://catalog.onliner.by/";
+    private final String login = "vitalyzenkou@gmail.com";
+    private final String password = "Qwerty123";
+    private WebDriver chromeDriver;
+
+    @BeforeClass
+    public void setup() {
+        chromeDriver = WebDriverFactory.getChromeDriver();
+        chromeDriver.get(url);
+    }
+
+    @Test
+    public void productAddedToCart() {
+        CatalogOnliner catalog = new CatalogOnliner(chromeDriver);
```

Comment:
in PageObject philosophy you should name `Page` classes with `Page` ending
`CatalogPage` `LoginPage` etc

Reply:
fix